package.json

```diff
@@ -124,10 +124,10 @@
     "proxy:build": "cd proxy && cargo build --all-features --all-targets",
     "proxy:build:release": "cd proxy && cargo build --release --all-features --all-targets",
     "proxy:clean": "cd proxy && cargo clean",
-    "proxy:start": "cd proxy && cargo build --bin git-remote-rad && cargo run -- --registry=emulator",
-    "proxy:start:test": "cd proxy && cargo build --bin git-remote-rad && cargo run -- --registry=emulator --test",
-    "proxy:start:ffnet": "cd proxy && cargo build --bin git-remote-rad && cargo run -- --registry=rpc.ff.radicle.network",
-    "proxy:start:devnet": "cd proxy && cargo build --bin git-remote-rad && cargo run -- --registry=35.241.138.91",
+    "proxy:start": "cd proxy && cargo build --bin git-remote-rad && cargo run",
+    "proxy:start:test": "cd proxy && cargo build --bin git-remote-rad && cargo run -- --test",
+    "proxy:start:ffnet": "cd proxy && cargo build --bin git-remote-rad && cargo run",
+    "proxy:start:devnet": "cd proxy && cargo build --bin git-remote-rad && cargo run",
```

Comment:
Shouldn't we remove these altogether?

Reply:
indeed :mag:

Reply:
And a bunch more actually. Thanks for your attentive look :ok_hand:

Reply:
I think he meant to delete the `ffnet` and `devnet` comamnds alltogether.